### ISSUE TITLE
sab 3.0.0b1 - remove job from history does not delete files

### ISSUE THREAD

**Issue by thezoggy** [sabnzbd/sabnzbd#1470]
TL;DR; sonarr removed release from sab but sab did not clean up the files

sab 3.0.0b1  win64 binary on Windows 7

sonarr had problem importing file due to episode title being TBA. Then later a proper release came out and sab downloaded that release as well. So now this morning I see two releases for the same episodes in limbo waiting to be imported (as sonarr still has TBA for the episode title). I went ahead and told sonarr to remove the non-proper release so it would remove it from history+delete files. 

sonarr:
```
20-5-29 11:25:09.8|Trace|HttpClient|Res: [GET] https://192.168.0.2:8080/api?mode=history&name=delete&del_files=1&value=SABnzbd_nzo_rruf1lde&apikey=(removed)&output=json: 200.OK (92 ms)
20-5-29 11:25:09.8|Trace|HttpClient|Response content (16 bytes): {"status": true}
```
I see sab got the request:
```
2020-05-29 11:25:09,803::INFO::[database:245] [sabnzbd\api.py.del_hist_job] Removing job SABnzbd_nzo_rruf1lde from history
```
The release was in fact removed from the history.. but the files were NOT deleted

**Comment by Safihre** [comment]
Was this a failed job? We never supported removing Completed jobs from history and deleting files.

**Comment by thezoggy** [comment]
was not a failed job. 
gotcha, well that explains why i find random stuff left in sab download folder never cleaned up over these years with automation apps :/

**Comment by Safihre** [comment]
The incomplete folder is in SABnzbd's control, so deleting files is a predictable action. But the Complete folder is out of our control due to user's moving files manually or using automation.
So supporting deleting files there would be unpredictable and something we just can't support.
